src/__tests__/github.mapping.test.ts

```diff
@@ -0,0 +1,95 @@
+import { mapProject, mapProjects } from '../mapper/github.mapping';
+
+describe('mapProject', () => {
+  it('should map a repository object to a project object', () => {
+    const repository = {
+      name: 'test-repo',
+      description: 'This is a test repository.',
+      url: 'https://github.com/test/test-repo',
+      language: 'Python',
+    };
+    const languages = ['Python', 'JavaScript'];
+    const contributors = [
+      { name: 'johndoe', avatarUrl: 'https://example.com/avatar.png' },
+    ];
+
+    const project = mapProject(repository, languages, contributors);
+
+    expect(project).toEqual({
+      name: 'test-repo',
+      description: 'This is a test repository.',
+      url: 'https://github.com/test/test-repo',
+      language: 'Python',
+      contributors: [
+        { name: 'johndoe', avatarUrl: 'https://example.com/avatar.png' },
+      ],
+      languages: ['Python', 'JavaScript'],
+    });
+  });
+});
+
+describe('mapProjects', () => {
+  it('should map an array of repository objects to an array of project objects', () => {
+    const repositories = [
+      {
+        name: 'test-repo1',
+        description: 'This is the first test repository.',
+        url: 'https://github.com/test/test-repo1',
+        language: 'Python',
+      },
+      {
+        name: 'test-repo2',
+        description: 'This is the second test repository.',
+        url: 'https://github.com/test/test-repo2',
+        language: 'JavaScript',
+      },
+    ];
+    const repositoriesLanguages = {
+      'test-repo1': ['Python', 'JavaScript'],
+      'test-repo2': ['JavaScript', 'CSS', 'HTML'],
+    };
+    const repositoriesContributors = {
+      'test-repo1': [
+        { name: 'janedoe', avatarUrl: 'https://example.com/avatar1.png' },
+        { name: 'sarahdoe', avatarUrl: 'https://example.com/avatar2.png' },
+      ],
+      'test-repo2': [
+        { name: 'johndoe', avatarUrl: 'https://example.com/avatar.png' },
+      ],
+    };
+    // const repositoriesLanguagesArray = Object.values(repositoriesLanguages);
```

Comment:
I subscribe to the ideology that code should be self documenting (e.g. very little comments). I only write comments for context related stuff.

e.g. There is a bug with the library, the code must do this.

Or something like that.

However, this is dead code. Please remove it.